upkie/envs/wrappers/servos_actions.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Inria
+
+import gymnasium
+from gymnasium import spaces
+
+class DirectTorqueControl(gymnasium.Wrapper) :
```

Comment:
Could you make one class per file? (This helps exploring the code, e.g. for newcomers.)

Reply:
```suggestion
class TorqueControl(gymnasium.Wrapper) :
```
Shorter?